test/views/ViewSpecBase.scala

```diff
@@ -67,6 +67,9 @@ trait ViewSpecBase extends SpecBase with BaseSelectors {
   def assertRenderedById(doc: Document, id: String): Assertion                  =
     assert(doc.getElementById(id) != null, "\n\nElement " + id + " was not rendered on the page.\n")
 
+  def assertRenderedByClass(doc: Document, className: String): Assertion =
+    assert(doc.getElementsByClass(className) != null, "\n\nElement " + className + " was not rendered on the page.\n")
```

Comment:
this is never null because `getElementsByClass` return an Collections which defaults to empty please fix it by changing like here https://github.com/hmrc/claim-a-trust-frontend/pull/132/files